### DISCUSSION TITLE
Change / Deactivate Timeout Time for TCP-Tunneling

### DISCUSSION THREAD

**Discussion by MikeT-KBU** [rustdesk/rustdesk#9862] [Feature Request] [answered]
We asked if it is possible to Change Timeout time (30 seconds) for the TCP-Tunneling.

Answer from Support: "Yes, it is 30 seconds, this is intended. There is no way yet, we may add an option for you, please submit a feature request here, we will follow up."

Here is the Feature Request for it.

**Comment by rustdesk** [comment, marked as answer]
Not ready and planned yet, PR is welcome

**Reply by rustdesk** [reply]
The timeout is set both on controlled side and controlling side, it is not just a simple one side option.
